ui/narrative/methods/run_QUAST_app/display.yaml

```diff
@@ -24,6 +24,10 @@ parameters :
         ui-name : Assemblies
         short-hint : Assembly objects to be analyzed by QUAST.
 
+    force_glimmer:
+        ui-name : Force Gene Calls For Large Assemblies
```

Comment:
It's not just large assemblies, it's if the total number of bases across all assemblies is large - not sure if this is accurate

Reply:
@dcchivian Should we just put "Force Gene Calls"?